modules/ui/src/app/services/state.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class StateService {
+  focusFirstElementInMain() {
+    const firstControl: HTMLElement | null = window.document.querySelector(
+      '#main button:not([disabled="true"]), ' +
+        '#main a:not([disabled="true"]), #main table'
+    );
+
+    if (firstControl) {
+      firstControl.focus();
+    }
+  }
+}
```

Comment:
I don't think we need to push this changes to dev as we didn't do anything related to this on the release v1.1.2. So, the dev branch has more correct last changes.
Let discuss.